### ISSUE TITLE
Clicking on a Radio station brings up the MPD servers menu when it was already selected

### ISSUE THREAD

**Issue by gerroon** [segler-alex/RadioDroid#789]
Hi

I am not sure if this is by design or not so I am reporting as a bug.

I have multiple MPD servers setup. All work well individually but one thing that is a bit sucky is that everytime I click on a radio station, I get the MPD servers menu even when an MPD server was already actively selected. 

Add couple MPD sevrers
Enable MPD 
Click on a radio
It brings up the MPD menu
Choose one of the servers
Listen for couple secons
While playing click on a radio station
It brings up the MPD menu, even though  we already  selected one for playing
Now you need to stop the active MPD server first
Then Play again to be able to play the radio you have been trying to play.

Basically one has to first stop the MPD server to be able to play another radio.

**Comment by werman** [comment]
> I get the MPD servers menu even when an MPD server was already actively selected.

That's the main point which was changed, now there is no "main" server - there are just playback "targets" if I can name them that way. 

E.g. you can have such targets: RadioDroid, external player (e.g. VLC on your device), several MPD servers and ChromeCast. This menu is an attempt to unify all of these to offer a transparent way to work with them.

> Now you need to stop the active MPD server first
> Then choose an MPD play button to be able to play the radio you have been trying to play.

I agree that this is not an optimal flow.

I have two thoughts on this:
1) There should be a way to play new station in one click after bringing up this menu.
2) I'm not sure how it can look but maybe we can indeed select the main playback target and not bring up selection menu when station is clicked.
On the second thought, there is a option menu for each mpd entry in the menu, we can add "Play here by default" option and mark the entry.

In my opinion 1) has a higher priority since it's a straight forward bad UX. 

Anyway, thank you for bringing this issue up.

**Comment by gerroon** [comment]
Why not just have a setting like "use active Mpd server for playback". If ticked, radios just play to that MPD server. 

It would be very nice if the user does not need to mess with additional fialogs or clicks if the user already selected an MPD server for playing.

Your 2nd suggestion sounds more likeable to me.

**Comment by werman** [comment]
"Active MPD server" is a bit fuzzy concept since you can play stations on any number of MPD servers simultaneously, however having something similar like "Play via last MPD server by default" could be good enough.  What do you think?

**Comment by gerroon** [comment]
That sounds great. Thanks for considering.

**Comment by Johan-EU** [comment]
The same menu is shown after clicking on a station when Chromecast is enabled. All apps I use with Chromecast just play on the device selected after clicking on the Chromecast icon. Showing another selection menu is confusing.

**Comment by werman** [comment]
I think the option could be expanded to "Play via last selection by default" (Wording probably could be improved) and placed inside the dialog to be visible.

**Comment by STPKITT** [comment]
My idea would be to incorporate a certain action when long-clicking the play button. For example long-clicking could be made configurable to either always play via MPD or always play on main MPD-server in case the user has setup more than one MPD-instance in Radiodroid.
